<launch>

  <!-- <rosparam command="load" file="$(find control)/config/gps_config.yaml" /> -->
  <!-- <node pkg="control" type="bridge" name="localization_bridge" output="screen"> </node> -->
  <rosparam command="load" file="$(find mpc)/scripts/config/mpc_config25.yaml" />
  <rosparam command="load" file="$(find mpc)/scripts/config/mpc_config_park.yaml" />
  <arg name="sign" default="false"/>
  <arg name="ekf" default="false"/>
  <arg name="lane" default="false"/>
  <arg name="name" default="car1"/>
  <arg name="vref" default="0.25"/>
  <arg name="x0" default="11.88"/>
  <arg name="y0" default="1.91"/>
  <arg name="yaw0" default="0"/>
  <arg name="real" default="false"/>
  <arg name="subModel" default="false"/>
  <arg name = "path" default="path1"/>
  <param name="pathName" value="$(arg path)"/>
  <param name="vrefName" value="$(arg vref)"/>

  <node pkg="mpc" type="controller" name="$(anon mpc_controller)" output="screen"> 
    <param name="sign" value="$(arg sign)"/>
    <param name="ekf" value="$(arg ekf)"/>
    <param name="lane" value="$(arg lane)"/>
    <param name="name" value="$(arg name)"/>
    <param name="vref" value="$(arg vref)"/>
    <param name="x0" value="$(arg x0)"/>
    <param name="y0" value="$(arg y0)"/>
    <param name="yaw0" value="$(arg yaw0)"/>
    <param name="real" value="$(arg real)"/>
    <param name="subModel" value="$(arg subModel)"/>
  </node>
</launch>
